test.py

```diff
@@ -846,6 +846,8 @@ async def run(self, options: argparse.Namespace) -> Test:
             self.args.insert(0, "--manager-api={}".format(manager.sock_path))
 
             try:
+                # Note: start manager here so cluster (and its logs) is availale in case of failure
+                await manager.start()
```

Comment:
Is this manager.start() function idempotent, i.e., if the manager is already started, it doesn't start again?

Reply:
I'll add this.

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/510215d79a70dcaa70a4cdc414555694d8e5ba51